src/components/Stories/TrendingStories.js

```diff
@@ -121,9 +121,9 @@ class TrendingStories extends Component {
                   ]
                 }}
               >
-                <Media heading className="trending">
+                <CardHeader className="stories-cardheader">
```

Comment:
You can use bootstrap class for typography here instead of creating new class:
`<CardHeader className="h5">`